server/queue/queue.php

```diff
@@ -39,24 +39,32 @@ function getAppointments($date, $siteId) {
 	try {
 		$canViewClientInformation = $USER->hasPermission('view_client_information');
```

Comment:
Don't you want this information showing on the appointment management page too? This is in the queue rn which is good, but I figured we'd want it in the appointment management page too since I think that's more often used to view appointment info?

Reply:
Good point, I will look into adding that